core/tests/protractor_utils/forms.js

```diff
@@ -89,10 +90,10 @@ var GraphEditor = function(graphInputContainer) {
     clearDefaultGraph: async function() {
       var deleteButton = graphInputContainer.element(
         by.css('.protractor-test-Delete-button'));
-      await deleteButton.click();
+      await action.click('Test Delete Button', deleteButton);
       // Sample graph comes with 3 vertices.
       for (var i = 2; i >= 0; i--) {
-        await vertexElement(i).click();
+        await action.click(`Vertex Element ${i}`, vertexElement(i));
       }
     },
     expectCurrentGraphToBe: async function(graphDict) {
```

Comment:
Below, please add a waitFor before the `.count()` call. The waitFor should only execute when `edgesList.length > 0`

Reply:
Done!